src/SDK.ts

```diff
@@ -190,13 +197,19 @@ export default class SDK implements ISDK {
     if (!requestId) {
       requestId = uuidv4();
     }
-    let endpoint = `${this.omnichannelConfiguration.orgUrl}/${OmnichannelEndpoints.LiveChatGetChatTokenPath}/${this.omnichannelConfiguration.orgId}/${this.omnichannelConfiguration.widgetId}/${requestId}`;
-    const headers: StringMap = Constants.defaultHeaders;
+    let endpoint;
+    const headers: StringMap = Constants.defaultHeaders;  
+
+    if(this.liveChatVersion==LiveChatVersion.V2){
```

Comment:
Use `==` instead of `===`

nit: space before and after `===`

Reply:
nit: space after `if` and before `{`

Reply:
I'd avoid having an `else` as follow

```ts
let endpoint = `${this.omnichannelConfiguration.orgUrl}/${OmnichannelEndpoints.LiveChatGetChatTokenPath}/${this.omnichannelConfiguration.orgId}/${this.omnichannelConfiguration.widgetId}/${requestId}`; // use default endpoint

if (this.liveChatVersion === LiveChatVersion.V2) { // update endpoint only if it's LiveChatV2
    endpoint =`${this.omnichannelConfiguration.orgUrl}/${OmnichannelEndpoints.LiveChatv2GetChatTokenPath}/${this.omnichannelConfiguration.orgId}/${this.omnichannelConfiguration.widgetId}/${requestId}`;
}
```

Reply:
replace "==" with "===" and added spacing